GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -268,6 +268,9 @@ public FormCommit([NotNull] GitUICommands commands, CommitKind commitKind = Comm
             commitAuthorStatus.ToolTipText = _commitCommitterToolTip.Text;
             skipWorktreeToolStripMenuItem.ToolTipText = _skipWorktreeToolTip.Text;
             assumeUnchangedToolStripMenuItem.ToolTipText = _assumeUnchangedToolTip.Text;
+            stageToolStripMenuItem.Text = toolStageItem.Text;
```

Comment:
This is temporary for release/3.4, translation string to be used in later releases (translation will be affected which we do not want now)